git/homework.py

```diff
@@ -48,7 +57,11 @@ def multiple_ints(first_value: int, second_value: int) -> int:
     Returns:
         Product of elements
     """
-    pass
+    if type(first_value) is int and type(second_value) is int:
+        product = first_value * second_value
+        return product
```

Comment:
could combine line 62 with current.
You don't need to create separate variable for return